client/src/components/CardPile/CardPile.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react'
+import { CSSTransition, TransitionGroup } from 'react-transition-group'
+
+import Card from '../Card'
+import { Card as CardType } from '../../types/Card'
+import './CardPile.scss'
+
+export interface CardPileProps {
+  cards: {
+    type: CardType
+    id: string
+  }[]
```

Comment:
Consider making a type for this? `Array<Card>` would be more readable.

Reply:
So are we team `Array<Card>` or team `Card[]` ?

Reply:
Team `Array<Card>` it is !